src/scheduler/server_info.c

```diff
@@ -223,8 +223,12 @@ query_server(status *pol, int pbs_sd)
 		return NULL;
 	}
 
+	sched = pbs_statsched(pbs_sd, NULL, NULL);
+	while(sched && strncmp(PBS_DFLT_SCHED_NAME, sched->name, PBS_MAXSCHEDNAME)) {
```

Comment:
Why are you only looking for the default scheduler?  Shouldn't you be looking for sc_name?  This is the policy object that will be used throughout the scheduling cycle.  A multisched should use its own policy object, not the default schedulers.

Reply:
Thank you very much for catching this @bhroam. It looks like we added this as placeholder in the very beginning days of  multisched where the core interface was not developed at that time and forgot to replace it.  @Shrini-h  Since the core interface is now checked in, we should use sc_name only as suggested by Bhroam.  Also It would be better if we add couple of PTL test cases(There might already be test cases existing for these already just need to modify them to be suitable for Multisched) covering the following attributes for Multisched and add it our test suites TestMultipleSchedulers or TestSchedulerInterface whichever is fine.  Please let me know in case if you need any help on this.
* do_not_span_psets, only_explicit_psets
* preempt_targets_enable
* job_sort_formula_threshold
* opt_backfill_fuzzy

Also please run all the test cases from the following test suites in addition to what you are testing.
* TestMultipleSchedulers 
* TestSchedulerInterface

Reply:
@suresh-thelkar I will change  the default scheduler to sc_name.  But I dont think PTL changes on Multisched will fall in the scope of this Bug or commit.

Reply:
@Shrini-h 
As you are touching the  Multisched code it is always better to run the existing Multisched test cases as I mentioned above just to make sure that nothing is broken.  
Regarding the couple of new test cases that I have suggested, it is up to you whether to add as part of this bug or they can be added as part of QA Verification.

Reply:
@suresh-thelkar , I agree to run the Multisched test cases here.  But lets take the PTL changes outside the scope from here

Reply:
Now considering the impact of this new found bug I have filed a new bug https://pbspro.atlassian.net/browse/PP-1238